std/os/netbsd.zig

```diff
@@ -673,7 +673,7 @@ pub fn fstat(fd: i32, buf: *c.Stat) usize {
     return errnoWrap(c.fstat(fd, buf));
 }
 pub const iovec = extern struct {
-    iov_base: [*]u8,
+    iov_base: ?*c_void,
```

Comment:
I'm pretty sure this should stay as [*]u8. It's the same ABI. The question is whether or not the address 0x0 is valid, for whether to have a `?`.